app/dashboards/work_process_dashboard.rb

```diff
@@ -50,6 +52,6 @@ class WorkProcessDashboard < Administrate::BaseDashboard
   COLLECTION_FILTERS = {}.freeze
 
   def display_resource(work_process)
-    work_process.title
+    work_process.title.truncate(MAX_TITLE_LENGTH)
```

Comment:
I see the `title` field is not NOT NULL on the db, which means some work processes will have a nil title. We'll have an undefined method error then, should we protect ourselves against it?

Reply:
oops, ok, there's a presence validation on the model, so we're fine